src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -289,17 +295,31 @@ export default class PhotoField extends React.Component {
   }
 
   render() {
-    const file = this.props.formData || {};
+    const {
+      uiSchema,
+      formData,
+      formContext,
+    } = this.props;
+    const { file } = formData || {};
+    const onReview = formContext.reviewMode;
     const errorMessage = file.errorMessage || this.state.errorMessage;
-    const smallScreen = isSmallScreen(this.state.windowWidth);
-    const fileTypes = this.props.uiSchema['ui:options'].fileTypes;
+    const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(formContext.pageTitle);
```

Comment:
Can you explain some of the changes in here, aside from the review page check on line 345? Some of these I think would conflict with changes I made in the screen reader PR. I'm not sure what changes we would need to make in here other than the review mode switch.